multiwallet_test.go

```diff
@@ -0,0 +1,35 @@
+package dcrlibwallet_test
+
+import (
+	"os"
+	"testing"
+	"testing/quick"
+
+	. "github.com/raedahgroup/dcrlibwallet"
+)
+
+// canUseDir returns true if the program can create the directory
+// if it doesn't exist or the directory is empty if it exists.
+// If will delete the directory if it is empty.
+func canUseDir(directory string) bool {
+	if os.MkdirAll(directory, os.ModePerm) == nil {
+		return os.Remove(directory) != nil
```

Comment:
Still won't work. `os.Remove` will be nil if the directory is empty. Doesn't make the directory unusable. My recommendation:
```suggestion
		return true
```

Reply:
It passes...

Reply:
Why wouldn't an empty dir be usable?

Reply:
It is usable. But following your code, an empty directory will return false here because `os.Remove` will be nil.

Reply:
It can't be empty if it got here, `os.MkdirAll` creates it?

Reply:
`os.MkdirAll` creates an empty directory.

Reply:
Oh yeah, was confused for a sec